lib/govuk_publishing_components/components/related_navigation_sidebar_helper.rb

```diff
@@ -0,0 +1,140 @@
+require 'govuk_navigation_helpers/content_item'
+require 'govuk_navigation_helpers/grouped_related_links'
+
+class RelatedNavigationSidebarHelper
```

Comment:
Now `related_navigation_sidebar_helper` and `related_navigation_helper` are in the same place, it might be worth renaming these to make it clear what the difference is between them. Same goes for the related test files.

Reply:
Just having a think about this.  From what I understand `RelatedNavigationSidebarHelper` helps construct the links that are passed in to the related-navigation component, so we could rename to something like `RelatedNavigationLinksHelper`?  What do you think?

Reply:
Kind of, I think it's more about retrieving the data and putting it in a format that can be passed to the related navigation component

Reply:
I think we should lose the word `Sidebar` from however this is named. It infers position and this varies across devices.